src/core/client/framework/lib/bootstrap/createManaged.tsx

```diff
@@ -149,50 +140,48 @@ function createManagedCoralContextProvider(
 
     // This is called every time a user session starts or ends.
     private clearSession = async (nextAccessToken?: string | null) => {
-      // Clear session storage.
-      this.state.context.sessionStorage.clear();
```

Comment:
If we don't clear the session storage, then all the comment drafts will stay after logout and can be exposed to the next user who logs in.